tests/src/test/java/ca/bc/gov/open/jagefilingapi/qa/cucumber/Options/RunCucumberTest.java

```diff
@@ -9,7 +9,7 @@
          features = {"src/test/java/features"},
          glue ={"stepDefinitions"},
          monochrome = true,
-         tags = "@backend",
+        // tags = "@backend",
```

Comment:
do we need this?

Reply:
Could remove it for this one but will be uncommented in the next PR's to run specific tests only.